backend/services/api-test/src/test/java/io/metersphere/api/controller/ApiCalculateTest.java

```diff
@@ -227,10 +227,10 @@ public void calculateTest() throws Exception {
         Assertions.assertEquals(apiCoverageDTO.getApiCaseCoverage(), CalculateUtils.reportPercentage(apiCoverageDTO.getCoverWithApiCase(), apiCoverageDTO.getAllApiCount()));
 
         Assertions.assertEquals(4, apiCoverageDTO.getCoverWithApiScenario());
-        Assertions.assertEquals(12, apiCoverageDTO.getUnCoverWithApiScenario());
+        Assertions.assertEquals(16, apiCoverageDTO.getUnCoverWithApiScenario());
         Assertions.assertEquals(apiCoverageDTO.getScenarioCoverage(), CalculateUtils.reportPercentage(apiCoverageDTO.getCoverWithApiScenario(), apiCoverageDTO.getAllApiCount()));
 
-        Assertions.assertEquals(10, apiCoverageDTO.getCoverWithApiDefinition());
+        Assertions.assertEquals(6, apiCoverageDTO.getCoverWithApiDefinition());
         Assertions.assertEquals(10, apiCoverageDTO.getUnCoverWithApiDefinition());
         Assertions.assertEquals(apiCoverageDTO.getApiCoverage(), CalculateUtils.reportPercentage(apiCoverageDTO.getCoverWithApiDefinition(), apiCoverageDTO.getAllApiCount()));
 
```

Comment:
Code is fine and doesn't have any potential problems or issues that require adjustment. It's well written to achieve its specified goal of calculating APIs that were covered during test execution using the provided data. No changes are necessary in terms of code readability, format consistency, or structure alignment with current standards. The tests seem robust enough based on your specifications without additional adjustments needed at this time.

No optimization required for the given implementation due to it already being designed optimally.